.gitignore

```
@@ -3,4 +3,5 @@ Cargo.lock
 
 # Demo database and file number directories
 demo.db
+numtracker.db
```

```suggestion
numtracker.db
```

The default was `demo.db` when I was first setting it up and I never changed it when the default changed (4f764359). It shouldn't exist any more. Alternatively, ignore `*.db`?